Google-Summer-of-Code-2023.md

```diff
@@ -697,6 +699,49 @@ Additionally, finish adding [computer-aided translation (CAT)](https://docs.goog
 **Useful resources:**
 
 
+### 2.3. Improvements for translation reviewer experience
```

Comment:
Can we link the PRD (https://docs.google.com/document/d/1_vvewEwviQjtPvZy9yo74eCcRTQNcQKyCgdHNpyWNWw/edit?usp=sharing) under "Useful resources" for project 2.1?

Reply:
done